docs/demos/create-br1-linux-bridge.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkState
+metadata:
+  name: node01
+spec:
+  desiredState:
+    interfaces:
+      - name: eth1
```

Comment:
Please include nmstate description field stating what is each interface good for

Reply:
Done